src/main/java/seedu/address/logic/commands/ListAnimalCommand.java

```diff
@@ -0,0 +1,24 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.model.AnimalModel.PREDICATE_SHOW_ALL_ANIMALS;
+
+import seedu.address.model.AnimalModel;
+
+/**
+ * Lists all animals in the catalog to the user.
+ */
+public class ListAnimalCommand extends AnimalCommand {
+
+    public static final String COMMAND_WORD = "list";
+
+    public static final String MESSAGE_SUCCESS = "Listed all persons";
```

Comment:
```suggestion
    public static final String MESSAGE_SUCCESS = "Listed all animals";
```

Just a minor mistake!

Reply:
Oh right thanks!